client/my-sites/site-indicator/style.scss

```diff
@@ -105,6 +105,7 @@
 
 	// Links within the action message
 	a:link,
+	a:hover,
```

Comment:
I think this will still not take place because of the existing `.button.is-borderless.is-scary:hover` rule that turns the color red on hover. Do we need some more specificity?

Reply:
This specificity seems adequate, though I suppose that the stylesheets could be imported differently, and there's no harm in ensuring it's more specific so that it's never overridden. I've made that commit now. :)

<img width="562" alt="Screenshot 2020-01-21 at 19 06 07" src="https://user-images.githubusercontent.com/43215253/72834648-2b1d2700-3c81-11ea-802a-7e819629412f.png">